### PR TITLE
Add kubectl plugin to blue/green deployment on k8s

### PR DESCRIPTION
**Pull request by afshin-paydar**

### CODE REVIEW

**Comment by 4p00rv** [comment]
- In the script, we're just switching back and forth between blue and green. This is confusing from an external user point of view. Better to use the application version instead of blue/green.
- We can't have two versions running at the same time to avoid resource allocation wastage. At any point, we will only be using one version so the other should be shut down after a successful switchover. The reverts will require another release in any case.

File: kubectl-deploy/bin/kubectl-deploy [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+set -o nounset
+set -o errexit
+set -o pipefail
+
+DEFAULT_NAMESPACE="default"
+DEFAULT_TAG="latest"
+DEFAULT_TIMEOUT=120
+API_SERVER="127.0.0.1:8443"
+CERT="ca.crt"
+DOCKER_REPO="docker.io/phantasm"
```
Comment (4p00rv) [inline kubectl-deploy/bin/kubectl-deploy]: We shouldn't be using default here. Instead it is better to fail without proceeding any of the steps if the value is not present.

Comment (afshin-paydar) [reply kubectl-deploy/bin/kubectl-deploy]: Removed optional parameters.

**Comment by afshin-paydar** [comment]
- It should switch back of forth between blue and green and k8s admin can get the live environment by running below command:
```
$ kubectl get service production-deriv-com -o=jsonpath='{.spec.selector.env}'
$ kubectl get service staging-deriv-com -o=jsonpath='{.spec.selector.env}'
```
- It scale down passive environment now by `--replica 0` and scale up the live environment

**Pull request closed without merging**